docs/HTTP-batchsink.md

```diff
@@ -14,6 +14,9 @@ Properties
 
 **batchSize:** Batch size. Defaults to 1. (Macro enabled)
 
+**Json Batch Key** Optional key to be used for wrapping json array as object " +
```

Comment:
I could not understand what this meant until I saw the code. It would be good to include an example.

Reply:
Sure 👍 
> Example - If batch size is 2 and json batch key is "data", then the payload will be
`{"data": [{"key":"val"}, {"key":"val"}]}` instead of `[{"key":"val"}, {"key":"val"}]`